### PR TITLE
Better 404.html

### PR DESCRIPTION
**Pull request by 1337god**
I've made a better 404 page for you guys.

### CODE REVIEW

File: error/404.html [hunk 1]
```diff
@@ -2,12 +2,17 @@
 <html>
 <head lang="en">
   <meta charset="UTF-8">
-  <title></title>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge, chrome=1">
```
Comment (franzliedke) [inline error/404.html]: Do we really need this here?

File: error/404.html [hunk 2]
```diff
@@ -2,12 +2,17 @@
...
 </head>
 <body>
 
-<h1>404 Not Found</h1>
-<p>Looks like this page could not be found.</p>
+<h1>Error 404</h1>
+<p>We are looking for your requested page: <code id="uri"></code> but it seems that it doesn’t exist or it has been moved. Sorry!</p>
 
+<script>
+	document.getElementById("uri").innerHTML = "/" + (location.pathname+location.search).substr(1);
```
Comment (franzliedke) [inline error/404.html]: I would prefer for this URL to be generated on the server side. Our [`HandleErrors` middleware](https://github.com/flarum/core/blob/7837fff10762f25886cfa83a8b1bb9c0626f84c5/src/Http/Middleware/HandleErrors.php) could replace some simple strings such as `{{url}}` in the HTML that it reads from the templates.

**Comment by franzliedke** [comment]
Thanks for this suggestion! I like the idea. It would be nice if you could replace the JavaScript with some server-side code, though. :)

File: error/404.html (hunk 1 above)
Comment (1337god) [inline error/404.html]: because I will still make a better template, that's why I put it there.
Example: https://thevidme.club/fgdgdg

Comment (franzliedke) [inline error/404.html]: But for which feature is this needed?

**Comment by tobscure** [comment]
Our error pages have since been redone in b7c1cc5ceffb36bc990919bd749050f49f39554b. Thanks anyway for your PR!

**Pull request closed without merging**